src/blob/index/bptree/tests.rs

```diff
@@ -65,6 +65,10 @@ impl Into<usize> for KeyType {
     }
 }
 
+fn create_io_driver() -> IoDriver {
+    IoDriver::new_sync()
```

Comment:
I think it's better to create `IoDriver` here with `new()`, so that tests with `aync-io-rio` enabled use asynchronous I/O. Although previously tests were running in synchronous mode, now it is better to make it possible to run them with async mode too